README.md

```diff
@@ -428,14 +428,22 @@ Once installed, execute `py.test -vv` in the test directory to run all tests.
 We've added an installation file to make it easier to run tests and develop on
 Mac OSX. To install the dependencies on Mac OSX, run `install-mac-dev.sh`
 instead of `install.sh` and `install-dev.sh`.
-This will download Firefox ESR into the current folder, move geckodriver
+This will install Python packages in a local Python 3.6 virtualenv, 
+download Firefox Nightly into the current folder, move geckodriver
```

Comment:
We are actually not downloading Nightly. We are downloading an Unbranded build of the latest release.

Reply:
Of the latest general release or ESR release?

Reply:
I see it's the latest release - Fx66